vectorlm/utils/save_utils.py

```diff
@@ -270,9 +259,33 @@ def load_scheduler(
         rank: The worker's rank.
 
     """
-    sched_name = f"scheduler_rank{rank}.bin"
+    sched_name = "scheduler.bin"
     input_scheduler_file = os.path.join(input_dir, sched_name)
-    print(f"Loading scheduler state from {input_scheduler_file}")
+    if rank == 0:
```

Comment:
We should think about a master rank logger if we are using these conditionals for logging. We don't need to address it now.